swap/src/monero.rs

```diff
@@ -99,6 +99,20 @@ impl Amount {
         self.0
     }
 
+    pub fn max_bitcoin_for_price(&self, ask_price: bitcoin::Amount) -> Result<bitcoin::Amount> {
+        let piconero_minus_fee = self.as_piconero().saturating_sub(MONERO_FEE.as_piconero());
+
+        if piconero_minus_fee == 0 {
+            bail!("The current Monero balance is smaller than the basic locking fee");
```

Comment:
do you want me to return something like this?
```suggestion
             if piconero_minus_fee == 0 {
            return bitcoin::Amount::ZERO;
```

Reply:
Yep, that is better!